test/storage-luatest/storage_persistent_names_test.lua

```diff
@@ -0,0 +1,118 @@
+local t = require('luatest')
+local vtest = require('test.luatest_helpers.vtest')
+local vutil = require('vshard.util')
+
+local group_config = {{engine = 'memtx'}, {engine = 'vinyl'}}
```

Comment:
The `engine` param is not used. I suggest to drop it then. Given that you don't use any spaces here, could also drop `memtx_use_mvcc_engine`.

Reply:
Dropped `group_config`